.github/workflows/build_hub.yml

```diff
@@ -5,26 +5,38 @@ on:
     branches:
       - master
       - production
-    paths:
-      - 'prime-router/**'
-      - '.github/workflows/build_hub.yml'
   push:
     branches:
       - master
-    paths:
-      - 'prime-router/**'
-      - '.github/workflows/build_hub.yml'
 
 env:
   # These are for CI and not credentials of any system
   DB_USER: prime
   DB_PASSWORD: changeIT!
 
 jobs:
+  pre_job:
+    name: Pre Job
+    runs-on: ubuntu-latest
+    outputs:
```

Comment:
Strange that the step title in the build is dorny/paths etc.  I can't see a place here to change that.  Oh well.